ibis/expr/scope.py

```diff
@@ -159,7 +212,7 @@ def merge_scopes(
 
 
 def make_scope(
-    op: Node, result: Any, timecontext: Optional[TimeContext] = None
+    op: Node, timecontext: Optional[TimeContext], value: Any
```

Comment:
i would not above making `timecontext` and `value` keyword only here, e.g add a `*` in the signature, then you don't accidently set one